ui/app/services/version.js

```diff
@@ -48,7 +48,14 @@ export default class VersionService extends Service {
   }
 
   get hasSecretsSync() {
-    return this.features.includes('Secrets Sync');
+    const isEnterprise = this.isEnterprise;
```

Comment:
As mentioned in my comment above, this logic is mostly copy/pasted from the flags service. The biggest change here is there's no check for activation and subsequently nav permissions. Neither of these felt relevant to answering "does this feature exist for this cluster" which is (now) what this check is responsible for.

Anytime `hasSecretsSync` is referenced if a permissions or a activation check is also necessary that happens as well. This made it much easier to see gaps in logic (like we weren't previously checking nav permissions for HVD clusters - is this intentional or accidental?)